Modules/Core/Common/test/itkNumericTraitsTest.cxx

```diff
@@ -268,14 +266,14 @@ CheckSignedAndIntegerTraitsForComplexTypes(const char * const name)
   }
 
   // IsSigned same for complex and basic types??
-  if (itk::NumericTraits<T>::IsSigned != itk::NumericTraits<typename itk::NumericTraits<T>::ValueType>::IsSigned)
```

Comment:
This line should not be changed.

Reply:
This must be changed or lines 68 and 69 need to be resored.